packages/backend/tests/commands.spec.ts

```diff
@@ -60,741 +56,80 @@ describe("commands unit tests", () => {
-    //     .expects("withProgress")
-    //     .withArgs({
-    //       location: nsVsMock.testVscode.ProgressLocation.Notification,
-    //       title: messages.verify_cf_connectivity,
-    //       cancellable: false,
-    //     })
-    //     .throws(new Error("re-authenticate"));
-    //   vscodeWindowMock
-    //     .expects("withProgress")
-    //     .withArgs({
-    //       location: nsVsMock.testVscode.ProgressLocation.Notification,
-    //       title: messages.verify_cf_connectivity,
-    //       cancellable: false,
-    //     })
-    //     .resolves({ user: "bag023" });
-    //   // cmdLogin
-    //   vscodeWindowMock
-    //     .expects("showInputBox")
-    //     .withExactArgs({ prompt: messages.enter_user_email, ignoreFocusOut: true })
-    //     .resolves("user-mail");
-    //   vscodeWindowMock
-    //     .expects("showInputBox")
-    //     .withExactArgs({ prompt: messages.label_enter_password, password: true, ignoreFocusOut: true })
-    //     .resolves("psd");
-    //   vscodeWindowMock
-    //     .expects("withProgress")
-    //     .withArgs({ location: nsVsMock.testVscode.ProgressLocation.Window, title: messages.loggin_in })
-    //     .resolves();
-    //   //
-    //   vscodeWindowMock
-    //     .expects("withProgress")
-    //     .withArgs({ location: nsVsMock.testVscode.ProgressLocation.Window, title: messages.getting_orgs })
-    //     .throws(error);
-    //   vscodeWindowMock.expects("showErrorMessage").twice().resolves();
-    //   await commands.cmdCFSetOrgSpace();
-    // });
-
-    it("ok:: endPoint, org, space are provided", async () => {
-      vscodeWindowMock
-        .expects("withProgress")
-        .withArgs({
-          location: nsVsMock.testVscode.ProgressLocation.Notification,
-          title: messages.verify_cf_connectivity,
-          cancellable: false,
-        })
-        .resolves({ data: {} });
-      vscodeWindowMock.expects("showInformationMessage").withExactArgs(messages.success_set_org_space).resolves();
-      vscodeWindowMock
-        .expects("withProgress")
-        .withArgs({ location: nsVsMock.testVscode.ProgressLocation.Window, title: messages.set_org_space })
-        .resolves();
-      await commands.cmdCFSetOrgSpace();
-    });
-  });
+  describe("cmdLogin", () => {
+    const target = new cfView.CFTargetTI({ label: "target", isCurrent: true, isDirty: false });
+    const parent = new cfView.CFFolder("parent", target);
+    const node = new cfView.CFLoginNode(parent);
 
-  describe.skip("cmdSelectSpace", () => {
-    afterEach(() => {
-      sandbox.restore();
-    });
+    it("ok:: triggered from targets tree", async () => {
+      const json = {
+        Target: "my-endPoint",
+        SpaceFields: {
+          Name: "space",
+        },
+        OrganizationFields: {
+          Name: "org",
+        },
+      };
+      cfLocalUtilsMock.expects("cfGetConfigFileJson").resolves(json);
+      const ap = "https://api.cf.sap.hana.ondemand.com:8090";
```

Comment:
confidential data using is prohibited during testing. change it to some dummy url

Reply:
done